src/libs/Navigation/AppNavigator/Navigators/RightModalNavigator.js

```diff
@@ -26,7 +26,7 @@ function RightModalNavigator(props) {
         <NoDropZone>
             {!isSmallScreenWidth && <Overlay onPress={props.navigation.goBack} />}
             <View style={styles.RHPNavigatorContainer(isSmallScreenWidth)}>
-                <Stack.Navigator screenOptions={RHPScreenOptions}>
+                <Stack.Navigator screenOptions={RHPScreenOptions(styles)}>
```

Comment:
shouldn't `RHPScreenOptions(styles)` be memoized?

Reply:
I'm not certain but I think Rory mentioned about memoization around those functions passing styles because of frequent updates he noticed, and passing those to the navigator will refresh all screens I believe.